app/build.gradle

```diff
@@ -198,6 +198,7 @@ dependencies {
     implementation 'androidx.constraintlayout:constraintlayout:2.0.4'
     implementation 'androidx.core:core-ktx:1.3.2'
     implementation 'androidx.documentfile:documentfile:1.0.1'
+    implementation 'androidx.fragment:fragment-ktx:1.3.4'
```

Comment:
Why did you need to add a new library?

Reply:
We were only using a transitive version of the Fragment library, and that version was 1.2.4. We need to specify using Fragment version 1.3.4 to support the ActivityResult APIs.